concordium-cis2/src/lib.rs

```diff
@@ -31,6 +31,9 @@
 //! When `u256_amount` feature is enabled the type [`TokenAmountU256`] is defined
 //! and implements the [`IsTokenAmount`] interface.
 #![cfg_attr(not(feature = "std"), no_std)]
+
+pub mod cis2_client;
```

Comment:
```suggestion
pub use cis2_client::Cis2Client;
```

There is little reason to just expose the submodule.

Reply:
done